interview-questions.md

```diff
@@ -8,48 +8,47 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
Remember these questions are your way to practice that conversation with the interviewer. Engaged the interviewer by not defining the term but including some examples. Example are like a bull horn that shout you have used this tool before.